### PR TITLE
Add unit test for handlers funcs GetBeer and AddBeer

### PR DESCRIPTION
**Pull request by vhugo**
First time contributing with a PR to an open project, so I am not sure what should be in the message, but here it goes something.

This is a resolution for issue #9 

To improve testing experience and coverage I have made small changes to `main.go`, `data.go` and `handlers.go`.

In the `main.go` I have added an initialization function to populate beers and reviews before main() and I also set `router` variable to be accessible across the package, so I could use it in the tests.

In the `data.go` I've changed the behavior of functions that populate data to actually return the data instead of assigning it directly to the exported variables. Also to be easily used in the tests.

And at last, the changes on `handlers.go` was more to satisfy the tests by adding the new beer data to the exported Cellar variable.

### CODE REVIEW

**Comment by vhugo** [comment]
Due to the recent merge GetBeer() has been implemented so I've added a test for it. ;-)

**Comment by katzien** [comment]
Thanks @vhugo - will take a look at this tomorrow and digest it properly - getting late now! Sorry for the delayed response!

File: handlers.go [hunk 1]
```diff
@@ -66,6 +66,9 @@ func AddBeer(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 	if err != nil {
 		http.Error(w, "Bad beer - this will be a HTTP status code soon!", http.StatusBadRequest)
 		return
+	} else {
```
Comment (katzien) [inline handlers.go]: Whoops, haha, good catch! Can you remove the else block though and put it below the if statement? The if err != nil will return if there's an error, so no need for the else here :)

**Comment by vhugo** [comment]
@katzien added the changes as requested. 👍

**Pull request merged**